python/test/unit/test_diffusion.py

```diff
@@ -351,31 +350,59 @@ def simulate_and_test_diffusion(
             delta=self.dev * sV_tot_max_expected,
         )  # max_{t}(s⋅V) [direct]
 
+        # additional tests for the case that there is only one segment of fixed radius
+        if num_segs == 1:
+            self.assertAlmostEqual(
```

Comment:
In the varying radius case: Can't we estimate the equilibrium concentration as:
$N = \sum\limits_{i\in C} V_i Xi(t=0)$ where $C$ is the set of CV indices
and 
$X_i(t\rightarrow\infty) = \frac{N}{V_i}$
with 
$X_i(t\rightarrow\infty) = X_j(t\rightarrow\infty)\quad\forall i,j\in C$?

Reply:
Yes! I've addressed this now, also see the comment below.